src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 13px;
+  height: 13px;
+  border: 1.5px solid #fff;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url("./images/star.svg");
+}
```

Comment:
All code above not needed. Look, you can move  `background-image: url("./images/star.svg");` to general `.stars__star` selector and this will be the default value for stars. And then, depending on the modifier, you make the star 'active', in principle, what you do below